.github/workflows/kanister-image-build.yaml

```diff
@@ -0,0 +1,39 @@
+name: build-kanister-image
+
+on:
+  pull_request:
+    branches:
+    - master
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: kanisterio/build
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: dorny/paths-filter@v2
+      id: filter
+      with:
+        filters: |
+          modified:
+            - 'docker/build/**'
```

Comment:
Make it unconditional for now, and instead make it an "on demand" workflow.

Reply:
fixed